full/Dockerfile

```diff
@@ -75,30 +75,41 @@ USER gitpod
 RUN sudo echo "Running 'sudo' for Gitpod: success"
 
 ### Go ###
-ENV GO_VERSION=1.11.2 \
+ENV GO_VERSION=1.12 \
     GOPATH=$HOME/go-packages \
     GOROOT=$HOME/go
 ENV PATH=$GOROOT/bin:$GOPATH/bin:$PATH
 RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -xzv \
     && go get -u -v \
-        github.com/acroca/go-symbols \
-        github.com/cweill/gotests/... \
-        github.com/davidrjenni/reftools/cmd/fillstruct \
-        github.com/fatih/gomodifytags \
-        github.com/haya14busa/goplay/cmd/goplay \
-        github.com/josharian/impl \
-        github.com/nsf/gocode \
```

Comment:
@JanKoehnlein Would the current go LS work without them or we need to keep them till VS Code extension is not deployed to production?